### ISSUE TITLE
speed of loading Campaign delta chart

### ISSUE THREAD

**Issue by spickering-git** [trendmanagement/new_website#14]
@DinamickaDevelopment  Can you speed up the load time of the Campaign delta chart please. If loading a long period of time it is taking too long.

![image](https://cloud.githubusercontent.com/assets/18487289/25493100/e9d830f2-2b29-11e7-83d9-e8d9cf6ee2ff.png)

**Comment by DinamickaDevelopment** [comment]
Yes, on the delivery stage I will minify all the js and css code and it should decrease loading time

**Comment by DinamickaDevelopment** [comment]
I optimized the chart. It should render faster now
